developer_tests/forward_operators/work/rttov_unit_test.csv

```diff
@@ -0,0 +1,2 @@
+<DART Obs Type>,<Platform ID>,<Satellite ID>,<Sensor ID>,<Sensor type>,<Coef file name>,<Channels to use list (optional)>
+EOS_2_AMSUA_TB,9,2,3,mw,rtcoef_eos_2_amsua.dat
```

Comment:
this file is created by the 'rttov_test.f90' program. The rttov library v 12.3 does not have a `rtcoef_eos_2_airs.*` coefficient file but it can be downloaded after an internet search. I found it at
https://nwpsaf.eu/downloads/rtcoef_rttov12/rttov7pred54L/

Reply:
the README.rst goes over what happens and why this file is created by rttov_test.csh
I still wonder if it is worthwhile to use it in the unit tests.

Reply:
There unit tests call module_initialize repeatedly, that is why the sensor is repeatedly added.